src/node_buffer.cc

```diff
@@ -1175,6 +1175,28 @@ void Swap64(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(args[0]);
 }
 
+bool FastIsUtf8(v8::Local<v8::Value>,
+                const v8::FastApiTypedArray<uint8_t>& buffer) {
```

Comment:
I believe `FastApiTypedArray` is on the way out? (#55452)

Reply:
We can move to Local<Value> once V8 removes it

Reply:
That moment is now: it is not present in V8 13.6, and #57753 is removing all remaining uses of it across the codebase.

Reply:
Ah, I'll update the implementation. Nice catch. Thanks @Renegade334